Library/Homebrew/dev-cmd/unbottled.rb

```diff
@@ -243,4 +256,53 @@ def output_unbottled(formulae, deps_hash, noun, hash, any_named_args)
 
     puts "No unbottled dependencies found!"
   end
+
+  def output_lost_bottles
+    # $ git log --patch -G\^\ \+sha256.\*\ sonoma: --since=@\{\'1\ week\ ago\'\}
+    git_log = %w[git log --patch]
+    git_log << "-G^ +sha256.* #{@bottle_tag}:"
+    git_log << "--since=@{'1 week ago'}"
```

Comment:
`--follow` will follow renames in case that's useful at all.

Reply:
The main problem with renames is the following:

Imagine we have a formula Alpha that is renamed to Beta later on. The platform bottle was removed in the Alpha formula, then the formula was renamed to Beta and then the bottle is restored. These changes happen in at least three separate commits.

I don't remember the formula name but I did find an example of this when looking at lost bottles for some old macOS version like big sur but that was before we scoped the search back to a week (I was looking 3 months back at the time). I could check if the formula has been renamed for each formula and use the current name but I don't know if it's worth the extra complexity. I'll give it a shot though.

Reply:
Ah, it's a one line change. It does add a second to the execution time on my machine from ~1.7 to ~2.7 seconds but that's not a big deal for a dev command and I'm sure it's faster on newer machines. No-brainer really.

Reply:
Handled in https://github.com/Homebrew/brew/pull/16115/commits/83598bb950bdb55b23232f7fe6767852aa856a23

Reply:
Thanks!